html/includes/api_functions.inc.php

```diff
@@ -1693,3 +1695,34 @@ function list_logs()
     $app->response->headers->set('Content-Type', 'application/json');
     echo _json_encode($output);
 }
+
+function validate_column_list($columns, $tableName)
+{
+    global $config;
+    $app = \Slim\Slim::getInstance();
+
+    $column_names = explode(',', $columns);
+    $db_schema = Symfony\Component\Yaml\Yaml::parse(file_get_contents($config['install_dir'] . '/misc/db_schema.yaml'));
+    $valid_columns = array();
+    foreach ($db_schema[$tableName]['Columns'] as $col) {
```

Comment:
You can use array_column()  (we have a library adding it for 5.3 and soon will be able to use the native implementation)